### ISSUE TITLE
Where did the "organize" step go in the overview graphics?

### ISSUE THREAD

**Issue by bfassbender** [ddd-crew/ddd-starter-modelling-process#62]
I love the new two refreshed diagrams
- https://github.com/ddd-crew/ddd-starter-modelling-process/blob/master/resources/ddd-starter-modeling-circular.svg
- https://github.com/ddd-crew/ddd-starter-modelling-process/blob/master/resources/ddd-starter-modeling-steps.svg

One thing I wonder about is where the "organize" part went? There is a step "categorize" no.
Organize is still part of the written description in https://github.com/ddd-crew/ddd-starter-modelling-process/blob/master/README.md, though.

I personally love the organize part because it focusses on bounded context relationships and looks at resulting team relationships.

**Comment by mploed** [comment]
I agree with @bfassbender ... Categorize should be changed back to Organize. Actually I was just about to open an issue with regards to exactly the same topic.

**Comment by svenackermann** [comment]
Maybe another issue in the picture related to that: "Categorize" and "Code" have exactly the same subtext. I would say for "Categorize" it should be "strategic" instead of "tactical".

**Comment by geovanneb** [comment]
Hi DDD Crew,

Thank you very much for the excellent guidelines.
I've recently noticed some inconsistencies between the DDD Starter Modeling Circular image and the descriptions in the README.md file.
Some of them was already pointed out by the other members, some still not addressed:

1. In the image, Step 1 is labeled “Align,” but in the README.md, it’s referred to as “Understand.”
2. The image labels Step 6 as “Categorize,” whereas the README.md uses “Organize.” Quick note: In the image, the text says, “Categorize your bounded contexts with tactical patterns.” However, it seems this step is more about organizing teams than categorizing contexts. Is that correct?
3. Lastly, in the README.md file the order of “Strategize” and “Connect” appears inconsistent. Should it be “Connect” first, followed by “Strategize”?

Thank you very much.

**Comment by javujavichi** [comment]
hellooo, I did the change, sorry the confusion

**Comment by bfassbender** [comment]
Thanks for looking into it. One last change needed, I think. 

The step is called organize not categorize. It then fits the chapters of the guide and it's a better fitting term as in "organize your teams around bounded contexts"

**Comment by mploed** [comment]
@javujavichi thank you so much for fixing this. I actually wanted to raise the same concern as @bfassbender. He is right. It should be called "Organize" and not "Categorize". I agree that the initial title "organize" was also a better fit and with "organize" your beautiful visualization is consistent with the textual content of the starter modeling process.

**Comment by javujavichi** [comment]
omg, so sorry, I'll do the change today night! 
Thank you very much :)
